### PR TITLE
refactor(replication-merger): upgrade MapColoniesJSONLogger version to 1.0.0

### PR DESCRIPTION
**Pull request by syncush**
related to #60

### CODE REVIEW

File: images/replication-merger/start.py [hunk 1]
```diff
@@ -350,24 +353,26 @@ def merge_job_loop(time_unit):
...
 
 if __name__ == '__main__':
-    # create a dir for the default log file location
-    os.makedirs('/var/log/osm-seed', exist_ok=True)
-    # pass service/process name as a parameter to JSONLogger to be as an identifier for this specific logger instance
-    log = JSONLogger('main-debug', additional_fields={'service': 'replication-merger'})
-    main()
\ No newline at end of file
+    base_log_path = os.path.join('/var/log', app_name)
+    service_logs_path = os.path.join(base_log_path, app_name + '.log')
```
Comment (melancholiai) [inline images/replication-merger/start.py]: Add the ".log" format as a constant variable

File: images/replication-merger/start.py [hunk 2]
```diff
@@ -350,24 +353,26 @@ def merge_job_loop(time_unit):
         log.info(f'state before merge job: { get_current_state_status(time_unit) }')
         log.info('running osmosis merge-replication-files command')
         run_command(f'{OSMOSIS_DIR} --merge-replication-files workingDirectory={time_unit.get_path()}',
-                    log.info,
-                    log.error,
+                    process_log.info,
+                    process_log.info,
                     handle_osmosis_exit_code,
-                    (lambda: log.info('Osmosis finished successfully.')))
+                    (lambda: log.info('{0} finished successfully.'.format(osmosis_name))))
```
Comment (melancholiai) [inline images/replication-merger/start.py]: along the script f-strings are being used

File: images/replication-merger/start.py [hunk 3]
```diff
@@ -350,24 +353,26 @@ def merge_job_loop(time_unit):
...
         log.info(f'state after merge job: { get_current_state_status(time_unit) }')
-        scheduale_next_merge(time_unit)
+        schedule_next_merge(time_unit)
 
 def main():
-    log.info('replication-merger container started')
+    log.info('{0} container started'.format(app_name))
```
Comment (melancholiai) [inline images/replication-merger/start.py]: same

File: images/replication-merger/start.py (hunk 2 above)
Comment (syncush) [reply images/replication-merger/start.py]: done

File: images/replication-merger/start.py (hunk 3 above)
Comment (syncush) [reply images/replication-merger/start.py]: done

File: images/replication-merger/start.py (hunk 1 above)
Comment (syncush) [reply images/replication-merger/start.py]: done

**Review by melancholiai — APPROVED** [review]

**Pull request merged**